spec/features/bulk_updates/bulk_updates_search_spec.rb

```diff
@@ -4,7 +4,9 @@
   before(:all) do
     # Create a few collections with unique attributes that we can search for
     2.times { |i| publish_draft(version: "nasa.001#{i}") }
-    3.times { |i| publish_draft(short_name: "nasa.002#{i}") }
+    3.times { |i| publish_draft(short_name: "nasa.002", version: "#{i}") }
+
+    wait_for_cmr
```

Comment:
I added this while I was working on fixing the tests. I think that there is some indexing that happens after publishing (at least in the real CMR) that might be async and sometimes slow. We have a comment about it in line 127 in the `get_revisions` method in `CollectionsController`.

I think the tests work fine (locally) without this though, do you think I should remove it?